backend/src/services/tests/muralcollection.test.ts

```diff
@@ -0,0 +1,152 @@
+import { database } from "../../config/database";
+import { MuralCollection } from "../../models/muralcollection.model";
+import { MuralCollectionService } from "../muralcollection.service";
+import { Mural } from "../../models/mural.model";
+import { Borough } from "../../models/borough.model";
+import { Artist } from "../../models/artist.model";
+
+beforeAll(async () => {
+  await Mural.belongsTo(Borough, {
+    foreignKey: { allowNull: false, name: "boroughId" },
+  });
+  await Mural.belongsTo(Artist, {
+    foreignKey: { allowNull: false, name: "artistId" },
+  });
+  await Mural.belongsToMany(MuralCollection, {
+    foreignKey: "muralId",
+    through: "murals_in_collection",
+    as: "collections",
+  });
+  await MuralCollection.belongsToMany(Mural, {
+    foreignKey: "collectionId",
+    through: "murals_in_collection",
+    as: "murals",
+  });
+});
+
+beforeEach(async () => {
+  await MuralCollection.sync({ force: true });
+  const params = {
+    name: "testmural",
+    boroughId: "1",
+    artistId: "1",
+    year: 1234,
+    city: "montreal",
+    address: "1234 street",
+    partners: ["partner 1", "partner 2"],
+    coordinates: {
```

Comment:
I merged master and added this coordinate field for you so I could try the tests with my new database. They all pass!